frontend/ui/elements/reader_menu_order.lua

```diff
@@ -216,6 +216,7 @@ local order = {
         "----------------------------",
         "ota_update", -- if Device:hasOTAUpdates()
         "help",
+        "search_menu",
```

Comment:
(I'd rather see it under `Help>` than alongside - but I understand you want to show off :)
This would make this menu 10 items, so we could move it after some time under Help when we need the 10th slot for a more valuable first level menu item.

Reply:
You might also argue it should be under search instead of help.

Reply:
We might, and my argument against that would be (from https://github.com/koreader/koreader/issues/9800#issuecomment-1319261649, as it was under `Search>` in the screenshot of #9800):
> (I might then request "Search menu entry" to be moved outside _the Search menu which contains only useful reading or word related search functions_ :) ie. in the Help> submenu.)

Reply:
Below or above quickstart guide?

Reply:
I would say below (first, you read the quickstart guide, then you use koreader, then you may need to search the menu :)